core/src/internal/rpc/mod.rs

```diff
@@ -3,25 +3,74 @@ use scaii_defs::protos::{MultiMessage, ScaiiPacket};
 use scaii_defs::protos::{ModuleInit, RpcConfig};
 use std::error::Error;
 use std::net::{IpAddr, SocketAddr};
-
 use websocket::client::sync::Client;
 use websocket::stream::sync::TcpStream;
 
 use scaii_defs::protos::endpoint::Endpoint;
-
+use std::process::Child;
```

Comment:
Instead, you can do `use std::process::{Child, Command};`

Reply:
ok